### DISCUSSION TITLE
Where should I load custom completions?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [ohmyzsh/ohmyzsh#11342] [Help] [answered]
I am trying to load completions for [Forc](https://fuellabs.github.io/sway/v0.31.1/introduction/installation.html#enable-tab-completion-for-bash-fish-zsh-or-powershell), which is like Cargo for the Sway programming language.

Unfortunately, Forc doesn't have an Oh My Zsh plugin. Where should I load its completions? I can think of two places:

1. In the `$ZSH_CUSTOM` directory.
2. In the `$HOME/.zshrc` file, near where the plugins are loaded.

**Comment by carlosala** [comment, marked as answer]
You need to add a file named `_forc` with the completion inside in a directory that is present in `$fpath`. Then you need to delete `$ZSH_COMPDUMP` file and do `omz reload`. After that, you should be good to go.
Seeing the documentation it would be `forc completions --shell=zsh > /path/that/is/in/fpath/_forc`

**Reply by PaulRBerg** [reply]
Great, thanks a lot Carlo.

I guess that I could follow their docs and put the completion in `~/.zfunc`, right?

**Reply by carlosala** [reply]
If that directory is on fpath, yas! Check it doing `echo $fpath | grep zfunc`. But you can also put that wherever!

**Reply by PaulRBerg** [reply]
Awesome, thanks for explaining.

**Comment by Tamarwish** [comment]
Hi

**Reply by Tamarwish** [reply]
How are you
